topics/assembly/tutorials/largegenome/tutorial.md

```diff
@@ -25,6 +25,8 @@ contributors:
 
 # Introduction
 
+*Note: We recommend running this tutorial on either the Galaxy Europe or Galaxy Australia servers. Other servers (such as Galaxy main) have not yet been configured fully for all the tools in this analysis.*
```

Comment:
If this note is specific to Smörgåsbord, I don't think we need to include it. Under every video we'll shown a list of which servers are supported (we check every tutorial against every tool version that's installed on each server.)

Reply:
Thanks @hexylena - sounds good, yes let's not include this.